packages/@okta/i18n/src/properties/login.properties

```diff
@@ -963,7 +963,7 @@ oie.registration.form.title = Sign up
 oie.registration.form.submit = Sign Up
 oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
-oie.user.profile.primary.email= Primary email
+oie.user.profile.primary.email= Email
```

Comment:
This may break the monolith selenium tests. Please create a downstream build to verify before merging.

Reply:
There's one mapping in `i18nTransformer` to this key - `enroll-profile.userProfile.email` - The related selenium tests (in monolith, if any) may break.

Reply:
Yes i will verify that first.